### DISCUSSION TITLE
is it possible to set Single Click to open files

### DISCUSSION THREAD

**Discussion by SebastianBugReporter** [elementary/triage#632] [Q&A] [answered]
Hello everyone,

I'm new here and I'm wondering if it's possible to set single click for opening files, like other distributions?

Thanks in advance

**Comment by jeremypw** [comment, marked as answer]
Sorry, Files now only supports a mixed-mode where folders open with a single-click (for navigation) and files are *selected* with a single-click but opened with double-click.  As this was arrived at after much "discussion" I am not sure there is an appetite to reintroduce multiple click modes with a setting. It may be possible to support e.g. open with single middle-click.

**Reply by vvvvvvitor** [reply]
Honestly, I don't understand the reasoning behind that decision, it's major inconsistency!

If you aren't going to support it, why even provide an option for it?

**Comment by SebastianBugReporter** [comment]
Please don't get me wrong, I don't want this as standard, it should just be offered as an option, like in other distributions, where you can choose whether you want to use single or double click

**Reply by jeremypw** [reply]
I understand, but whether or not it is standard it still requires a significant amount of extra code/complexity to maintain, which was the motivation for moving to a single mode in the first place. It cannot be provided by a third party plug-in or the like.  Sorry.

**Comment by SebastianBugReporter** [comment]
It's a shame, especially considering that in older versions you could freely choose how you wanted to open folders and files, i.e. I could set everything with one click or everything with a double click.
I would also like to say that everyone who uses single click normally uses it for everything, i.e. for folders and files, and those who use double click do it for both, I don't know anyone who prefers to open folders with one click and files with a double click.
But it is what it is, and someone must have thought of something when they changed it that way.

**Reply by jeremypw** [reply]
It is logical in the sense that a single-click results in an action *within the app* and a double-click results in action that takes you out of the app (launches another app). However, people do tend to prefer the ways they are used to.

**Comment by SebastianBugReporter** [comment]
Hello everyone,

Since over a year has passed and there's even a new version available, I wanted to gently inquire again whether there have been any changes regarding the single-click file opening feature.

Perhaps it's planned or possible to make this available as a feature that those who want it can use, but no one has to :-)

Many thanks for any information.
